src/gl.rs

```diff
@@ -248,6 +248,30 @@ pub fn bind_attrib_location(program: GLuint, index: GLuint, name: &str) {
     }
 }
 
+#[cfg(not(target_os="android"))]
+#[inline]
+pub fn get_uniform_block_index(program: GLuint, name: &str) -> GLuint {
+    unsafe {
+        ffi::GetUniformBlockIndex(program, CString::new(name).unwrap().as_ptr())
```

Comment:
This doesn't look like a safe usage of CString. The actual value being pointed to will go out of scope before GetUniformBlockIndex is invoked.